voice/bxml/verbs/speakSentence.md

```diff
@@ -253,7 +266,7 @@ Console.WriteLine(response.ToBXML());
 response = Bandwidth::Voice::Response.new()
 speak_sentence = Bandwidth::Voice::SpeakSentence.new({
     :sentence => 'Hello, you have reached the home of <lang xml:lang="es-MX">Antonio Mendoza</lang>.Please leave a message.',
-    :voice => "Sophia"
+    :voice => "sophia"
```

Comment:
are our voice case sensitive? If so, let's handle that in the SDK with a `.lowerCase()` or w/e for each of these sensitivities

Reply:
It is. I copy-pasted an example that has it uppercase and it failed the BXML validation. I'm not against going that route, but if we do I'd like to confirm with the voice team that all voices will be lowercase